frontend/src/store/auth/index.js

```diff
@@ -5,9 +5,9 @@ import CV from './CV'
 import { createProfile, updateProfile, getSelf } from '@/utils/api/auth'
 
 // TODO: more idiomatic way to do this? Router something?
-const _baseURL = `${window.location.protocol}//${window.location.host}`
-const _continueVerifyURL = `${_baseURL}/create-profile`
-const _continueResetURL = `${_baseURL}/login`
+const APP_URL = `${window.location.protocol}//${window.location.host}`
+const CONTINUE_VERIFY_URL = `${APP_URL}/create-profile`
```

Comment:
VUE_APP_URL? Why not?

Reply:
![screenshot from 2019-02-08 15-08-36](https://user-images.githubusercontent.com/4468620/52458459-91658280-2bb3-11e9-9281-6d789953b1b8.png)
![screenshot from 2019-02-08 15-08-53](https://user-images.githubusercontent.com/4468620/52458464-962a3680-2bb3-11e9-8945-65a5c4e2f45d.png)

Reply:
This is a meme